src/collection-view.js

```diff
@@ -369,7 +369,7 @@ Marionette.CollectionView = Marionette.View.extend({
 
     // Trigger `attach`
     if (canTriggerAttach && this._triggerAttach) {
-      nestedViews = [view].concat(view._getNestedViews());
+      nestedViews = [view].concat(_.result(view, '_getNestedViews') || []);
```

Comment:
Not sure what the advantage of _.result is. 

Also could the [] default go in getNestedViewd